### DISCUSSION TITLE
Logging Underlying Fiat Value for Outbound Transactions

### DISCUSSION THREAD

**Discussion by marcinx** [btcpayserver/btcpayserver#6477] [Ideas / Feature Requests]
In my jurisdiction (and many others too) it is necessary to keep a log of the underlying fiat value at the time of the sales event in BTC and also the time of the disposal of the BTC (e.g. payout to exchange, transfer to cold wallet, dividend payout, etc.)

As it stands the underlying fiat value at the sales time is logged in the invoice and can be exported via the legacy invoice report. 

It would be useful if all transactions that leave the wallet (e.g. via Wallet -> Send) log the underlying fiat value of the amount sent out in the store currency. Currently I can obtain the BTC/LTC amounts of outbound transactions but not the underlying fiat value, via Reporting -> Wallets. It would help if it would also display the store currency value.

There are more sophisticated methods to address accounting needs, e.g. calculating FIFO (first-in first-out) formulas etc. but I feel like simply logging the fiat price for outbound txs would be a great first step to facilitate accounting needs.

**Comment by dennisreimann** [comment]
@NicolasDorier I think it'd be really nice to offer this feature and track a set of configurable fiat pairs (e.g. USD + EUR) for transactions (incoming and outgoing). I really missed something like that when I initially set up my accounting.
